wire-ios/Wire-iOS/Sources/Managers/Image/ImagePickerManager.swift

```diff
@@ -96,23 +107,13 @@ class ImagePickerManager: NSObject {
             imagePickerController.cameraDevice = .front
             imagePickerController.modalTransitionStyle = .coverVertical
         case .photoLibrary, .savedPhotosAlbum:
-            if viewController.isIPadRegular() {
+            if viewController.isIPad() {
```

Comment:
`isIPadRegular` isn't sufficient. We need to use a popover on any iPad size

Reply:
I think correct would be to check if the split view controller's layout is collapsed, which is true for iPhones and for iPads when the window is resized so that the horizontal size class is compact.

Reply:
But won't that mean that it is possible to present this on iPad without using a popover? Won't that crash?

Reply:
I'll see if there is a iPad at work to test this with

Reply:
I tried if it still works with the full-screen requirement disabled
<img width="385" alt="Screenshot 2024-11-11 at 16 22 35" src="https://github.com/user-attachments/assets/652bd470-62a9-4841-93a9-bc818df8c718">
and resized the window so that the layout collapsed.
It looks like it does. 👍

Reply:
@caldrian You mean you tested there was no crash?

Reply:
No crash and the presentation seems to be correct.

Reply:
I just remembered this about `UIModalPresentationStyle.popover`:

> In iOS 13 and later, for horizontally or vertically compact environments, this option behaves the same as [UIModalPresentationStyle.formSheet](doc://com.apple.documentation/documentation/uikit/uimodalpresentationstyle/formsheet).

Perhaps this is what you meant @caldrian?

So we get the behavior we want just by checking for iPad. No need to worry about size class also.

![Simulator Screenshot - iPad (10th generation) - 2024-11-25 at 09 12 48](https://github.com/user-attachments/assets/c6706850-843f-4a8a-9cc8-0d8f832caf41)